core/dbt/context/base.py

```diff
@@ -687,6 +688,18 @@ def diff_of_two_dicts(
                 dict_diff.update({k: dict_a[k]})
         return dict_diff
 
+    @contextmember
+    @staticmethod
+    def md5(value: str) -> str:
```

Comment:
As discussed in https://github.com/dbt-labs/dbt-core/issues/6246 - let's come up with a different name for this (e.g. `local_md5` or `jinja_md5`), to avoid confusing users with the subtle distinction between `md5()` and `{{ md5() }}` in Jinja-SQL code.

Reply:
hey @jtcohen6 - pushed a fix